prober/derp.go

```diff
@@ -111,7 +125,7 @@ func WithSTUNProbing(interval time.Duration) DERPOpt {
 
 // WithTLSProbing enables TLS probing that will check TLS certificate on port
 // 443 of each DERP server every `interval`.
-func WithTLSProbing(interval time.Duration) DERPOpt {
+func WithTLSProbing(interval time.Duration, regions ...string) DERPOpt {
```

Comment:
Doesn't seem like regions is being used here:
```suggestion
func WithTLSProbing(interval time.Duration) DERPOpt {
```